tests/factories.py

```diff
@@ -1,6 +1,12 @@
 import factory
+from faker import Faker
+from faker.providers import BaseProvider
+
 from resolver.models import User, Substance
 
+fake = Faker()
+Faker.seed(0)
```

Comment:
Did you have to seed this for some reason?  Won't this yield the exact same results whenever we run the test suite?

Reply:
removed